### ISSUE TITLE
Android priority documentation may not be correct.

### ISSUE THREAD

**Issue by ggiordan** [phonegap/phonegap-plugin-push#2540]
With the move to FCM, there may have been a change in the value of "PRIORITY" in the message payload.

The push plugin documentation states (https://github.com/phonegap/phonegap-plugin-push/blob/master/docs/PAYLOAD.md#priority-in-notifications)
"Add a priority field in your notification. -2: minimum, -1: low, 0: default , 1: high, 2: maximum priority."

However, the FCM documentation states (https://firebase.google.com/docs/cloud-messaging/http-server-ref):
'priority |  Optional, string
Sets the priority of the message. Valid values are "normal" and "high."  On iOS, these correspond to APNs priorities 5 and 10.
By default, notification messages are sent with high priority, and data messages are sent with normal priority.
Normal priority optimizes the client app's battery consumption and should be used unless immediate delivery is required.  For messages with normal priority, the app may receive the message with unspecified delay.
When a message is sent with high priority, it is sent immediately, and the app can display a notification.

There are 2 things here that are operating, I have done some experimenting and seem to be true, but I'm happy to be corrected
1) priority should have either "high" or "normal" as a value
2) if you're sending only the data element (see the notification vs data discussion), and you want a timely delivery you need to set priority:'high' in your payload.

The bottom line is that to get timely delivery is when only sending a data element is to also have priority="high"

**Comment by macdonst** [comment]
@ggiordan the priority described in the docs refer to the priority of the notification in the notification shade. https://developer.android.com/reference/android/support/v4/app/NotificationCompat.Builder.html#setPriority(int)

Setting the priority outside of the `data` part of the payload works as normal.
